packages/provider/src/transcription-model/v1/transcription-model-v1.ts

```diff
@@ -0,0 +1,71 @@
+import { TranscriptionModelV1CallOptions } from './transcription-model-v1-call-options';
+import { TranscriptionModelV1CallWarning } from './transcription-model-v1-call-warning';
+
+/**
+Transcription model specification version 1.
+ */
+export type TranscriptionModelV1 = {
+  /**
+The transcription model must specify which transcription model interface
+version it implements. This will allow us to evolve the transcription
+model interface and retain backwards compatibility. The different
+implementation versions can be handled as a discriminated union
+on our side.
+   */
+  readonly specificationVersion: 'v1';
+
+  /**
+Name of the provider for logging purposes.
+   */
+  readonly provider: string;
+
+  /**
+Provider-specific model ID for logging purposes.
+   */
+  readonly modelId: string;
+
+  /**
+Generates an array of transcripts.
+   */
+  doGenerate(options: TranscriptionModelV1CallOptions): PromiseLike<{
+    /**
+Generated transcript as a string.
+The transcript contains the text that was transcribed from the audio.
+     */
+    transcript: {
+      text: string;
+      segments: Array<{
+        start: number;
+        end: number;
+        text: string;
+      }>;
+      language: string | undefined;
+      duration: number | undefined;
```

Comment:
what is the unit? can we include it in the var name to avoid confusion, e.g. `durationInSeconds`